modules/nativoBidAdapter.js

```diff
@@ -337,9 +345,13 @@ export const spec = {
             meta: {
               advertiserDomains: bid.adomain,
             },
+            mediaType: getMediaType(request.bidderRequest.bids[i]),
           }
 
           if (bid.ext) extData[bid.id] = bid.ext
+          if (bidResponse.mediaType === VIDEO) {
+            bidResponse.vastUrl = 'data:text/xml;charset=utf-8,' + encodeURIComponent(bid.adm)
```

Comment:
I'm torn about what to do with the VAST data. Should we set bid.adm to bidResponse.vastXML and let the publisher deal with having the cache setup in the config? The data coming from the ad server is technically XML and don't want to force it as a Url if we don't have to on our end. Thoughts?

Reply:
The sense I got from Aalok was we wanted to only serve the XML over a URI - but we can double check with him

Reply:
If that's the case, can we pull the **'data:text/xml;charset=utf-8,'** into the AdServer ... then if anything needs to change we can do it on the AdServer. I don't want to hardcode it on Prebid because it's difficult to get publishers to always upgrade to whatever the newest version of Prebid is containing the changes.

Reply:
@jsfledd to create a ticket for Aalok
I'll follow up and modify the code as needed.

Reply:
Completed: https://nativo.atlassian.net/browse/ADS-8099